internal/cli/apps.go

```diff
@@ -389,11 +404,17 @@ auth0 apps update <id> --name myapp --type [native|spa|regular|m2m]
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var current *management.Client
 
-			// Get app id
 			if len(args) == 0 {
-				if err := appID.Ask(cmd, &inputs.ID); err != nil {
+				err := appID.Pick(cmd, &inputs.ID, cli.appPickerOptions)
+				if err != nil {
 					return err
 				}
+
+				if inputs.ID == "" {
```

Comment:
Why the nil check if the options function already returns an error if there are no apps?